apps/backend/src/lib/external-db-sync-sequence-visibility.test.ts

```
@@ -0,0 +1,186 @@
+            tableName: "EmailOutbox",
+            tenancyId,
+            emailOutboxId: rowB.id,
+          });
+          await tx.emailOutbox.deleteMany({
+            where: { tenancyId, id: { in: [rowA.id, rowB.id] } },
+          });
+        });
+      } catch (error) {
+        cleanupError = error;
+      }
+      if (testError == null && cleanupError != null) throw cleanupError;
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)